tests/test_visitors/test_ast/test_classes/test_getter_setter.py

```diff
@@ -370,7 +380,7 @@ def test_class_attributes_getter_setter(
 @pytest.mark.parametrize('access', ['', '_', '__'])
 @pytest.mark.parametrize(('first', 'second', 'third'), [
     ('attribute', 'some', 'other'),
-    ('attribute', 'some', 'some'),
+    ('attribute', 'some', 'another'),
```

Comment:
Why did you change this line? Was it failing?
Can you please add `('attribute', 'some', 'some'),` as well?

Reply:
We use class_mixed template third and firth arguments are used as suffixes for methods. 
If I will return some, some then I will get violation.

Reply:
Got it! Thanks!